addons/web_tour/static/src/tour_service/tour_interactive.js

```diff
@@ -14,17 +14,18 @@ import { TourStep } from "./tour_step";
 
 export class TourInteractive {
     mode = "manual";
+    listeners = [];
+    anchorEls = [];
+    currentAction;
+    currentActionIndex;
+
     /**
      * @param {Tour} data
      */
     constructor(data) {
         Object.assign(this, data);
         this.steps = this.steps.map((step) => new TourStep(step, this));
```

Comment:
Why put the steps in the object, used anywhere ?

Reply:
this.steps already exists if Object.assign(this, data);